change/@fluentui-react-accordion-6c87e36a-041f-42c0-bf30-3921e7273454.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Bump Griffel dependencies",
```

```suggestion
  "comment": "chore: Bump Griffel dependencies",
```

nit: the same is for other entries in changelog